_drafts/A BLE Primer.md

```diff
@@ -0,0 +1,1202 @@
+---
+title: "A BLE Primer"
+author: mafaneh
+---
+
```

Comment:
This post would benefit from a table of content. You can generate one with: 
```
## Table of Contents
 
<!-- prettier-ignore -->
* auto-gen TOC:
{:toc}
```

You can read more about TOC generation in the [Kramdown docs](https://kramdown.gettalong.org/converter/html.html#toc)

Reply:
Done